curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-461-almost-pi.md

```diff
@@ -8,24 +8,15 @@ dashedName: problem-461-almost-pi
 
 # --description--
 
-Let `f(k, n)` = eᵏ/ⁿ - 1. `k` can be any non-negative integer.
-
-```js
-const f = (k, n) =>
-  Math.exp(k / n) - 1;
-```
+Let `f(k, n)` = $e^\frac{k}{n} - 1$. `k` can be any non-negative integer.
```

Comment:
Does the word "can" belong here?
```suggestion
Let `f(k, n)` = $e^\frac{k}{n} - 1$. `k` be any non-negative integer.
```

Reply:
The original description on FCC must've had a typo, because it doesn't really make sense. I noticed the original wording from project euler (https://projecteuler.net/problem=461) is `for all non-negative integers k`. Would that be better?

Reply:
I have updated my suggested change to make the official site's sentence.